### PR TITLE
[werft] fixes no-test flag

### LINKED ISSUE #6943: [werft] no-test flag is broken
### Bug description

[werft] no-test flag is broken

### Steps to reproduce

see https://github.com/gitpod-io/gitpod/pull/6937

### Workspace affected

_No response_

### Expected behavior

no run test

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by iQQBot**
## Description

[werft] fixes no-test flag

This PR is base on fa7aaddd, if run with test it will cause failed
see [build log](https://werft.gitpod-dev.com/job/gitpod-build-iqqbot-werft-no-test-flag-is-broken-6943.1/logs) ws-manager no test

/werft no-test
/werft no-preview

## Related Issue(s)

Fixes #6943 

## How to test

run `werft run no-test`

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

Associated issue: *#6943*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6944#issuecomment-980741798" title="Approved">iQQBot</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

Associated issue: *#6943*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by princerachit — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 57f420866996e64bd5b468c078f33592f3e825e9</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6944#issuecomment-980741798" title="Approved">iQQBot</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6944#pullrequestreview-820109574" title="Approved">princerachit</a>*

Associated issue: *#6943*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
